apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,31 @@
+import { IDetailQueryResult, ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import './transcribed-audio-detail.full-view.presenter.css';
+
+export const TranscribedAudioDetailFullViewPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
```

Comment:
Further, we should either make units part of the data model or else call the prop `lengthMilliseconds`.

Reply:
I've renamed it for now.  But called it lengthSeconds.

Reply:
Corrected this to be lengthMilliseconds on the view model